test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestMemoryMXBean.java

```diff
@@ -716,33 +719,35 @@ public final void testGetMBeanInfo() {
 		// Now make sure that what we got back is what we expected.
 
 		// Class name
-		AssertJUnit.assertTrue(mbi.getClassName().equals(mb.getClass().getName()));
+		AssertJUnit.assertEquals(mb.getClass().getName(), mbi.getClassName());
 
 		// No public constructors
 		MBeanConstructorInfo[] constructors = mbi.getConstructors();
 		AssertJUnit.assertNotNull(constructors);
-		AssertJUnit.assertTrue(constructors.length == 0);
+		AssertJUnit.assertEquals(0, constructors.length);
 
 		// One public operation (JLM) + five from CILM.
 		MBeanOperationInfo[] operations = mbi.getOperations();
 		AssertJUnit.assertNotNull(operations);
-		AssertJUnit.assertTrue(operations.length == 6);
+		AssertJUnit.assertEquals(6, operations.length);
 
 		// One notification
 		MBeanNotificationInfo[] notifications = mbi.getNotifications();
 		AssertJUnit.assertNotNull(notifications);
-		AssertJUnit.assertTrue(notifications.length == 1);
+		AssertJUnit.assertEquals(1, notifications.length);
 
 		// Print description and the class name (not necessarily identical).
 		logger.debug("MBean description for " + mb.getClass().getName() + ": " + mbi.getDescription());
 
 		// Eight attributes - some writable.
```

Comment:
This comment is stale; it's unclear when "eight" was the correct number. Suggest:
```java
		// attributes - some writable
```